src/MoBi.UI/MoBi.UI.csproj

```diff
@@ -525,12 +520,6 @@
     <Compile Include="Views\EditMoleculeBuilderView.Designer.cs">
       <DependentUpon>EditMoleculeBuilderView.cs</DependentUpon>
     </Compile>
-    <Compile Include="Views\EditNeighborhoodView.cs">
```

Comment:
This file was creating GridControl on the fly. Not used anywhere so I deleted it